test/solana-example/src/idl/whirlpool/types.ts

```diff
@@ -0,0 +1,288 @@
+import {Codec, struct, u8, u128, u64, bool, i128, fixedArray, address, unit, sum, u16, i32, ref} from '@subsquid/borsh'
+
+export type OpenPositionBumpsType = {
```

Comment:
> Why Type suffix?
To avoid name collisions, otherwise half of the generated vars will be highlighted as types and the other half will not

> Why type and not an interface?
Just a personal preference

Reply:
Interfaces are usually better, because

1. They imply less work for compiler
2. Hover/intellisense behaviour in some cases is better.

For example,

```ts
type Foo = {
    hello: number
    world: number
}

interface Bar {
    hello: number
    world: number
}

interface Baz {
    foo: Foo
    bar: Bar
}

let baz!: Baz
```

<img width="595" alt="image" src="https://github.com/subsquid/squid-sdk/assets/762157/0aff1e06-9eae-4b8d-9846-9ee6223cc3e8">


Regarding the Type suffix, I don't know what to say. 

As I never use rainbow color schemes, it annoys me. Also scale type system doesn't have it.

Perhaps, it makes sense to export just type part. Codecs themselves are not of any use for end user.

Reply:
Sometimes codecs are useful too

Anyway,I think this can be solved by answering the question "Should we transform type names to camelCase or should we keep thier names as much as posible?"